image-processing/cut_houses/cut_houses.py

```diff
@@ -242,9 +251,18 @@ def cut_mask_from_image(image:str|np.ndarray, mask:str|np.ndarray, output:str =
         # cv2.imshow('Cutout Image Combined', cutout_image)
         # cv2.waitKey(0)
 
-        if(output != None):
+        if output is not None:
             try:
-                cv2.imwrite(output, cutout_image)
+                # copy from core/preprocessing _save_image -> prevent import errors
```

Comment:
Same here, why not move load and save to own file and import twice and adjust __init_.py